docs/clusterpools.md

```diff
@@ -147,3 +147,81 @@ spec:
 ```
 
 **Note** When you use installConfigSecretTemplate you will most likely want to disable MachinePools, so that Hive does not reconcile away from the machine config specified in install-config.yaml
+
+## Time-based scaling of Cluster Pool
+
+You can use kubenetes cron jobs to scale clusterpools as per a defined schedule. Following are the sample yaml configurations to scale a clusterpool to size 0 at 23:45 (11:45 PM) every Friday. 
+
+```yaml
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  namespace: hive
+  name: scale-clusterpool
+rules:
+- apiGroups:
+  - hive.openshift.io
+  resources:
+  - clusterpools
+  verbs:
+  - get
+- apiGroups:
+  - hive.openshift.io
+  resources:
+  - clusterpools/scale
+  verbs:
+  - patch
+
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: scale-clusterpool
+  namespace: hive
+subjects:
+- kind: ServiceAccount
+  name: sa-scale-clusterpool
+  namespace: hive
+roleRef:
+  kind: Role
+  name: scale-clusterpool
+  apiGroup: rbac.authorization.k8s.io
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sa-scale-clusterpool
+  namespace: hive
+
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: scale-clusterpools
+  namespace: hive
+spec:
+  schedule: "45 23 * * 5"
```

Comment:
add a comment that describes this schedule. people can easily look up but i think having that is more readable.